examples/trials/mnist-hyperband/mnist.py

```diff
@@ -0,0 +1,231 @@
+"""A deep MNIST classifier using convolutional layers."""
```

Comment:
I can't see much difference with old-mnist example. Could we reuse the old one and add a config for hyperband?

Reply:
no. the STEPS in hyperparameters is very important.